src/main/java/spongebob/command/FindCommand.java

```diff
@@ -0,0 +1,50 @@
+package spongebob.command;
+
+import java.util.ListIterator;
+
+import spongebob.storage.Storage;
+import spongebob.storage.TaskList;
+import spongebob.task.Task;
+import spongebob.ui.Ui;
+
+
+/**
+ * Command to find keyword in task list
+ */
+public class FindCommand extends Command {
+
+    private String[] arguments;
+
+    public FindCommand(String[] arguments) {
+        this.arguments = arguments;
+    }
+
+    /**
+     * executes the command, prints out list of tasks that contains keyword
```

Comment:
Remember that description should be a sentence. i.e. To start with a capital letter and end with a punctuation mark.